tests/common/test_view_frustum.py

```diff
@@ -0,0 +1,77 @@
+
```

Comment:
This is not technically a unit test, right?

Reply:
Thanks for the review Ayush. I think it still qualifies as a unit test "smokescreen" because if the code crashes then the test fails